### PR TITLE
remove the parts of Kafka API, extend to support none kafka system

### PR DESCRIPTION
**Pull request by lqjack**

### CODE REVIEW

File: debezium-ddl-parser/src/test/resources/mysql/examples/bitrix_queries_cut.sql [hunk 1]
```diff
@@ -1,2006 +0,0 @@
-#begin 1
```
Comment (Naros) [inline debezium-ddl-parser/src/test/resources/mysql/examples/bitrix_queries_cut.sql:1]: Why was the DDL parser SQL examples that are used to test the DDL parser with removed?

**Review by Naros — CHANGES_REQUESTED** [review]
There is quite a lot to digest here and we'll try and give this a proper review next week, but in the meantime I did do a quick look over the changes and I've commented on a few changes that don't quite make sense.  Could you please take a look at those.  

Also, could you please update the commit message to start with the issue number "DBZ-3571" and please rebase your changes against the current master to resolve conflicts.

**Comment by lqjack** [comment]
@Naros thank you review the patch. I have reverted the developed code .

**Comment by gunnarmorling** [comment]
Hey @lqjack, before talking about specifics of the implementation, can we take a step back and clarify _what_ exactly you'd like to achieve here, and _why_ you'd like to do this? Once we have reached agreement on that goal and that it is a worthy one, let's then think about the _how_. Thanks a lot for providing some context!

**Comment by lqjack** [comment]
@gunnarmorling 

_why you'd like to do this?_

   1, We need the CDC feature as a standalone one.  In this way we can construct the Flink connector based on Debezium to support the CDC features without rework them. 

  2,  The current work is one the way to remove the Apache Kafka related API.  We want to provide the option for user to choose whether they need the feature based on the Kafka platform.

**Comment by lqjack** [comment]
@gunnarmorling 

_why you'd like to do this?_

   1, We need the CDC feature as a standalone one.  In this way we can construct the Flink connector based on Debezium to support the CDC features without rework them. 

  2,  The current work is one the way to remove the Apache Kafka related API.  We want to provide the option for user to choose whether they need the feature based on the Kafka platform.

**Comment by derekm** [comment]
@lqjack -- Have you seen https://github.com/ververica/flink-cdc-connectors ? It uses Debezium Engine and is very similar to Debezium Server (in fact, it would be cool to see Debezium Server improve such that Ververica's flink-cdc-connectors could be refactored into a proper Debezium Server sink).

**Comment by lqjack** [comment]
@derekm thanks the rely, I will check the repo.

**Pull request closed without merging**